civictechprojects/sitemaps.py

```diff
@@ -17,7 +17,8 @@ class SectionSitemap(Sitemap):
         str(FrontEndSection.FindProjects.value),
         str(FrontEndSection.Donate.value),
         str(FrontEndSection.ContactUs.value),
-        str(FrontEndSection.Companies.value)
+        str(FrontEndSection.Companies.value),
+        str(FrontEndSection.Privacy.value)
```

Comment:
Adding the new front end section (the new page) to sitemaps.py so our generated sitemap.xml is correct.